src/sentry/deletions/tasks/nodestore.py

```diff
@@ -254,9 +232,13 @@ def delete_dangling_attachments_and_user_reports(
     group ID, therefore there may be dangling ones after "regular" model
     deletion.
     """
+    # We don't want to fail the deletion task if we can't delete the attachments and user reports
```

Comment:
Will those get cleaned up somehow else if that happens?

Reply:
I assume the 90 days clean up.